apps/tlon-mobile/src/App.main.tsx

```diff
@@ -61,11 +59,19 @@ const App = ({
   channelId: notificationChannelId,
 }: Props) => {
   const isDarkMode = useIsDarkMode();
-  const tailwind = useTailwind();
+
   const { isLoading, isAuthenticated } = useShip();
   const [connected, setConnected] = useState(true);
   const { lure, priorityToken } = useBranch();
   const screenOptions = useScreenOptions();
+
+  const onboardingScreenOptions = {
+    ...screenOptions,
+    headerShown: false,
+    animation: 'simple_push' as const,
```

Comment:
Curious why we need animation settings here, since it doesn't seem like we're setting them on other stacks?

Reply:
No good reason, left it accidentally while I was playing around. Fixed.